README.md

```diff
@@ -12,11 +22,14 @@ Wire Client written in Java
     <dependency>
         <groupId>com.wire</groupId>
         <artifactId>helium</artifactId>
-        <version>1.3.0</version>
+        <version>x.y.z</version>
     </dependency>
 </dependencies>
 ```
 
+Create a `LoginClient`, passing a Java WS RS http client previously created, then obtain the users' token
```

Comment:
What does `Java WS RS` stands for? maybe we could write the full meaning so its easier for other outside (me included 😅 )

Reply:
I'm with you here, unfortunately it's not me being lazy, that's the actual specification name. Back in the day the Java community really loved abbreviations. I changed the comment a bit and also wrote Jersey client as an example to be more precise